tests/test_models.py

```diff
@@ -17,3 +24,62 @@ def test_uri_generation_guards_against_failure(self):
 
         item.full_clean(exclude={"list"})
         self.assertIsNotNone(item.uri)
+
+
+class ListItemGetChildUrisTests(TestCase):
+    def setUp(self):
```

Comment:
Would be nice for this to be setUpTestData and run only once.

Reply:
done!